docs/source/howto/plugin_codes.rst

```diff
@@ -332,6 +333,37 @@ With your ``calculations.py`` and ``parsers.py`` files at hand, let's register e
         Strictly speaking, ``aiida-diff-tutorial`` is the name of the *distribution*, while ``aiida_diff_tutorial`` is the name of the *package*.
         The aiida-core documentation uses the term *package* a bit more loosely.
 
+ * Or using ``pyproject.toml`` for new package:
```

Comment:
For a howto I would suggest to stick with one option.

The pyproject.toml is a little more verbose and I wonder whether we need to specify flit as the build backend here, but if the consensus is that this is the way to go, you can remove the `setup.py`.

Reply:
Thanks for the quick reply. I'd vote for only keep `pyproject.toml` since the cookie-cutter is already only has `pyproject.toml` and the 'plugin packaging section is changed to only conform with PEP 621 in this PR. 
Most of cases, beginner users would simply copy and paste the code to have a taste, and then for creating a new plugin, the cookie-cutter should be always the starting point.

Reply:
Yes, let's keep only the `pyproject.toml`.

As mentioned you can experiment with removing the flit stuff to keep things a bit more "minimal" but that's up to you.